src/rust/ensogl/lib/theme/src/lib.rs

```diff
@@ -101,105 +104,101 @@ macro_rules! define_theme {
 // === Light Theme & Modules ===
 // =============================
 
-define_default_theme! { light_theme
+define_default_theme! { __light_theme__
```

Comment:
Let's make it a normal name, but module private, and then reexport it in `builtin`

Same for dark theme

Reply:
Leaving it for now, as you cannot then make `light_theme` private and reexport it in sub-omdule. Let's talk about it again,